packages/block-editor/src/components/block-tools/zoom-out-mode-inserters.js

```diff
@@ -44,6 +46,8 @@ function ZoomOutModeInserters() {
 				getSettings().__experimentalSetIsInserterOpened,
 			selectedBlockClientId: getSelectedBlockClientId(),
 			hoveredBlockClientId: getHoveredBlockClientId(),
+			inserterSearchInputRef:
+				getSettings().__experimentalGetInserterSearchInputRef(),
```

Comment:
We may be able to make this a private setting. Hopefully we can avoid `__experimental`